feature-extensions/apigee/setup.py

```diff
@@ -1,11 +1,11 @@
-#!/usr/bin/env python2
 import sys
 import subprocess
 import argparse
 import os.path
+sys.path.append('../config')  # noqa: E402
```

Comment:
@rajeevr2715  Let's reorganize the modules as per the suggestion. We can push these improvements as part of next release.

Reply:
> Why is this here? For module imports? It shouldn't be necessary. Just organize the submodules according to python standards and properly reference them. Work with the best practice for the language, not against them.
> 
> For instance, to import a function named `scenario3` that resides in `installer/cli/scenario3cli.py` relative to the top-level script, just reference like so:
> 
> `from installer.cli.scenario3cli import scenario3`

@bleggett Since we are already inside the multiaccount folder to run the setup.py command, then the sys.path do not have the top folder "config".